configuration/override_dir_structure.rst

```diff
@@ -41,8 +41,10 @@ at your project root directory.
 Override the Cache Directory
 ----------------------------
 
-You can change the default cache directory by overriding the ``getCacheDir()``
-method in the ``Kernel`` class of your application::
+You have 2 ways to change the cache directory.
```

Comment:
Maybe we should remove this sentence. Counting the number of alternatives is something that quickly gets out of sync when we add a new alternative or remove an existing one.

Reply:
changed it, let me know